src/dcos_e2e/node.py

```diff
@@ -23,22 +23,25 @@ def __init__(
         self,
         public_ip_address: IPv4Address,
         private_ip_address: IPv4Address,
+        default_ssh_user: str,
         ssh_key_path: Path,
     ) -> None:
         """
         Args:
             public_ip_address: The public IP address of the node.
             private_ip_address: The IP address used by the DC/OS component
                 running on this node.
+            default_ssh_user: The default username to use for SSH connections.
             ssh_key_path: The path to an SSH key which can be used to SSH to
-                the node as the ``Cluster.default_ssh_user``.
+                the node as the `default_ssh_user` user.
```

Comment:
Double backticks around `default_ssh_user` for monospace - this is generated using Rst and single backticks are italics there.